src/java/detectors/cookie_httponly/CookieHttpOnly.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package detectors.cookie_httponly_rule;
+
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.Cookie;
+
+public class CookieHttpOnly{
+    // {fact rule=cookie-httponly@v1.0 defects=1}
+    public void CookieHttpOnlyNoncompliant(HttpServletResponse res) {
```

Comment:
method name should not start with a capital letter: `cookieHttpOnlyNoncompliant`

Reply:
Renamed the method name